Src/ILGPU.Tests/AtomicCASOperations.tt

```diff
@@ -7,7 +7,7 @@ using Xunit;
 using Xunit.Abstractions;
 
 <#
-var atomicTypes = DefaultIntTypes.Concat(FloatTypes.Select(t => (t.Item1, t.Item2))).ToArray();
+var atomicTypes = DefaultIntTypes.Concat(FloatTypes.Skip(1)).ToArray();
```

Comment:
ISSUE: Incorrectly using `FloatTypes.Skip(1)`.

This was intended to skip the FP16 float type, but it does not yet exist. Instead, it is incorrectly skipping FP32.

```suggestion
var atomicTypes = AtomicNumericTypes;
```